token-minter/sources/modules/token_components.move

```diff
@@ -201,6 +203,12 @@ module minter::token_components {
         property_map::remove(&authorized_borrow(creator, token).property_mutator_ref, &key);
     }
 
+    public fun disable_ungated_transfer<T: key>(owner: &signer, token: Object<T>) acquires TokenRefs {
```

Comment:
We can remove this. Unless we want to be able to disable and enable ungated transfer.

Reply:
it is unfortunately needed as when we pre-mint as creator, need to enable ungated transfer, and then disable it later if token is soulbound after user claims/mints.